src/planscape/planning/views.py

```diff
@@ -306,6 +377,16 @@ def update_scenario_result(request: HttpRequest) -> HttpResponse:
 # This returns the empty set if given a plan that has no scenarios,
 # a plan that isn't owned by the user, or a planID that doesn't exist.
 def list_scenarios_for_plan(request: HttpRequest) -> HttpResponse:
+    """
+    Lists all Scenarios for a Plan.
+    Requires a logged in user, as a scenario must be associated with a user's planning area.
+
+    Returns: a list of the scenarios for the user.  Can return the empty list.  Will return an empty list
+      if given a plan that has no scenarios, a plan that isn't owned by the user, or a plan ID that doesn't exist.
```

Comment:
', or if there is no existing plan associated with the given planning area ID' *

As another note, plan = planning area, correct? For a future edit, may be worth making the language consistent with one term or the other

Reply:
Updated comments and switched all references from "plan" to "planning_area".